HarpGuru/OverlayMenu/MenuOptions/PozitionOption/nudgeHarpStrataByPozition/index.test.ts

```diff
@@ -0,0 +1,41 @@
+import {HarpStrataProps, ApparatusIds, PozitionIds, PitchIds, ActiveIds, getHarpStrata} from 'harpstrata'
+
+import {DisplayModes} from '../../../../HarpFace'
+
+import { nudgeHarpStrataByPozition } from './index'
+
+const baseHarpStrataProps: HarpStrataProps = {
```

Comment:
I am tempted to move these objects to shared `testResources` folders, but the problem is that following this pull request there is going to be quite a bit of reorganisation work to do, so I think this might be wasted effort at this point. This should be noted as a task for that next job.

Reply:
Issue https://github.com/js-jslog/harpnative/issues/28 raised